src/app/containers/StoryPromo/index.test.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { render } from 'react-testing-library';
+import { shouldMatchSnapshot } from '../../helpers/tests/testHelpers';
+import deepClone from '../../helpers/json/deepClone';
+import StoryPromo from '.';
+
+let item;
+const compleateItem = {
```

Comment:
Typo here and below where it's used.
```suggestion
const completeItem = {
```

Reply:
Done :)